demos/manifest.json

```diff
@@ -34,5 +34,13 @@
             "To turn off the invoicing delay, run:",
             "   kubectl exec -n px-kafka $INVC_POD -c invoicing -- kill -USR2 $INVC_PID"
         ]
+    },
+    "px-finagle": {
+        "description": "Microservice demo that generates thriftmux traffic with finagle",
+        "instructions": [
```

Comment:
Should the instructions reference which scripts a user should look at to see the traced traffic?

Reply:
That's a great point and something I forgot to mention in the PR. We developed a pxl script for mux [here](https://github.com/SedarG/finagle-helloworld/blob/2a78cc9c48235bd09b160efe3ef0c7fc6844d399/finagle-helloworld/misc/mux.pxl.script). So I think we should add this script to the PR and update this description to point users there.

The script creates a table and shows mux's request type parameter, which is a int64 rather than it's human friendly string ([source](https://github.com/pixie-io/pixie/blob/503b456b446c52d03019b76b8f7ae80b0ca2dbbe/src/stirling/source_connectors/socket_tracer/protocols/mux/types.h#L36-L59)). We wanted to display the string in the resulting table to make it more visually appealing, but we struggled to perform that mapping of int64 to string in the script.

Do you know how we can do that translation in the script? We tried to create a dictionary whose keys are int64s and values are the string, but couldn't determine a way to do it. I'll need to refresh my memory of what we tried since it's been a while.

Reply:
It seems like carnot has dedicated functions for [kafka](https://github.com/pixie-io/pixie/blob/c08aaa2c53ce95ee40817acae3f662a95994f6fb/src/carnot/funcs/protocols/kafka.h#L29) and [mysql](https://github.com/pixie-io/pixie/blob/4020e0a06549cc79134c0f08b325c023815a2285/src/carnot/funcs/protocols/mysql.h) for doing this type of thing. Maybe that would be the most appropriate solution?

Reply:
It would be great to get your PxL script included in the folder of community scripts! Maybe add that in a separate PR to keep this one small? 

Yeah, if Kafka and MySQL use UDFs to get a human readable request type, then it probably makes sense to do the same here.

Reply:
The first step of that (creating the UDF) will be done in https://github.com/pixie-io/pixie/pull/609. I'll follow up with the future changes once that is merged.

Reply:
And the mux_data script will be added in #612.

Reply:
Updated this text to mention that the `px/mux_data` script should be used to visualize the demo.